docs/quickstarts/expo.mdx

```diff
@@ -144,7 +144,8 @@ The following example demonstrates an Expo layout that defines a custom token ca
 ```tsx {{ filename: "app/_layout.tsx" }}
 import * as SecureStore from 'expo-secure-store';
 import { ClerkProvider, ClerkLoaded } from "@clerk/clerk-expo"
-import { Slot } from "expo-navigation"
+// Slot is imported from expo-router
```

Comment:
Code comment here is not necessary - the import makes it clear that slot is imported from expo-router so we can simply update the import !

```suggestion
```